actions/setup/action.yml

```diff
@@ -72,9 +72,11 @@ runs:
       # greater than v0.5.0 and install it.
       # the install process, so check to see if we are running >=5
       MINOR=$(echo $tag | cut -d '.' -f 2)
-      INSTALL_TSA="false"
-      if [ ${MINOR} -ge 5 ]; then
-        INSTALL_TSA="true"
+      if [[ -z "${INSTALL_TSA}" ]]; then
```

Comment:
I wonder if this would be better aligned as an input rather than an env variable?

Reply:
i thought about that, but then i still need to pass the env var down to the release setup script, so i thought it would be easier to just use an env var

Reply:
Yeah, makes sense :)